src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -193,6 +213,7 @@ const Actions: React.FC<Props> = ({
         setModalError(null);
         setDate(dayjs());
         setDriverName("");
+        setSelected([]);
```

Comment:
So when the modal closes, we unselect all the parcels, is that right?

If I select a parcel, click Delete Parcel Request and close the modal, the parcels I'd selected are still selected. If I then click Delete Parcel Request again, it gives me an error saying I need to select at least one parcel.

_Should_ we really be deselecting all parcels whenever the modal closes? Or only when the delete request is successful? I don't think we should deselect them when we download shipping labels etc.

Reply:
So when the modal closes we do unselect all parcels in the selected state, but the corresponding checkboxes still appear ticked in the table (as I couldn't figure out how to untick them), which is why you experienced the above behaviour
Ok I think I agree that it would be best to only deselect if the delete request is successful. -
The reason I did it like this was because if we successfully delete the parcel, it becomes hidden by default, so the user would have to find it again in order to manually unselect it, otherwise that deleted parcel will be passed into any other actions as one of the selected parcels.
I will change the behaviour to only deselect if deletion successful. But I think we should also find a way to make the corresponding checkboxes appear unticked, as they should do, but I couldn't figure this out - maybe another ticket ?

Reply:
Now resolved aside from unselected rows still showing as ticked checkboxes